README.md

```diff
@@ -63,6 +82,68 @@ When you've set up your MCVE, run these statements again:
 mvn clean verify
 ```
 
+
+### Instructions when using Gradle
+
+First comment out all the subprojects that are not part of your MCVE in `settings.gradle.kts`: you usually work only on one subproject.
+
+Then make sure the following Gradle task runs succesfully:
+
+```
+./gradlew :<relevant subproject>:check
+```
+
+
+#### How to prepare your MCVE
+
+For your MCVE, you will have to adapt a few things, probably. This may include:
+
+- The Java version (only if the Java version is important for your MCVE):
+  - Go to the relevant subproject's build file (i.e. `$REPO_ROOT/<relevant subproject>/build.gradle.kts`) and add the following line (adapting the version numer) somewhere after the `import` statements:
+
+
+```kotlin
+java.toolchain.languageVersion = JavaLanguageVersion.of(17)
+```
```

Comment:
There are a ton of ways to do this, and they're language specific. This stuff shouldn't be described with words, but by example.

It's much better to have a default that already works, and then show the user where they can adapt stuff. I'll do this the day it really matters. Until then, let's just work with the defaults.